src/js/CheckboxTree.js

```diff
@@ -123,6 +123,7 @@ class CheckboxTree extends React.Component {
 
     toggleChecked(node, isChecked, noCascade) {
         if (node.children === null || noCascade) {
+            if (node.disabled) return;
```

Comment:
Would prefer this check to go down to where the children loop is, because the originator of the check (the parent) had to be enabled, so this check is slightly redundant for the root request.

I am also not a fan of shorthand `if` statements, so I would prefer the longer version with curly braces (actually surprised this is not enabled as an ESLint rule...I'll have to look into that).